wildfly/amq-broker-ssl/scripts/script.cli

```diff
@@ -0,0 +1,12 @@
+/subsystem=messaging-activemq/pooled-connection-factory=activemq-ra:write-attribute(name=user,value="${env.ARTEMIS_USER}")
+/subsystem=messaging-activemq/pooled-connection-factory=activemq-ra:write-attribute(name=password,value="${env.ARTEMIS_PASSWORD}")
+/subsystem=messaging-activemq/remote-connector=artemis:map-put(name=params, key=ssl-enabled, value=true)
+/subsystem=messaging-activemq/remote-connector=artemis:map-put(name=params, key=trustStorePath, value="${env.TRUST_STORE_FILENAME}")
+/subsystem=messaging-activemq/remote-connector=artemis:map-put(name=params, key=trustStorePassword, value="${env.TRUSTSTORE_PASSWORD}")
+/subsystem=messaging-activemq/remote-connector=artemis:map-put(name=params, key=verify-host, value=false)
```

Comment:
I am not sure about this, but shouldn't the client verify the server (host) it is connecting to?

Reply:
The explanation makes sense, thanks. I agree with this configuration.
Also I see a comment from @ehsavoie here: https://stackoverflow.com/questions/77841066/how-to-configure-the-wildfly-messaging-activemq-subsystem-with-jms-to-work-over, which definitely seems related.
@ehsavoie we'd appreciate your input greatly here, WDYT?

Reply:
Maybe you should use the ssl context attribute there and set up the ssl context in elytron. that would be a nicer way to set up things

Reply:
@ehsavoie Thank you for the suggestion ... is there any example app/repo I can copy from?